### PR TITLE
Update additional nested build and test dependencies

### PR DESCRIPTION
**Pull request by jasonkatonica**
Additional updates are required for build and test packages maven-reporting-impl and maven-core. These packages have been updated to the latest levels available.

Signed-off-by: Jason Katonica <katonica@us.ibm.com>

### CODE REVIEW

**Review by taoliult — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -409,6 +409,26 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-checkstyle-plugin</artifactId>
             <version>3.4.0</version>
+           <exclusions>
```
Comment (KostasTsiounis) [inline pom.xml:412]: Indentation is a bit off.

**Review by KostasTsiounis — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jasonkatonica) [reply pom.xml]: I updated the indentation on this line. It was missing a space to line up with the lines above.

**Review by KostasTsiounis — APPROVED** [review]
LGTM

**Pull request merged**